src/getstream.js

```diff
@@ -26,7 +26,7 @@ function connect(apiKey, apiSecret, appId, options) {
    * @example <caption>where streamURL looks like</caption>
    * "https://thierry:pass@gestream.io/?app=1"
    */
-  if (process && process.env && process.env.STREAM_URL && !apiKey) {
+  if (typeof process !== 'undefined' && process.env && process.env.STREAM_URL && !apiKey) {
```

Comment:
could you add a note for angular specific?

Reply:
checked again and we have this check in many other places so seems fine without note.